pkgs/applications/editors/rstudio/default.nix

```diff
@@ -73,24 +53,39 @@ let
     hash = "sha256-lZnZvioztbBWWa6H177X6rRrrgACx2gMjVFDgNup93g=";
   };
 
-  description = "Set of integrated tools for the R language";
 in
-(if server then stdenv.mkDerivation else mkDerivation)
-  (rec {
-    inherit pname version src RSTUDIO_VERSION_MAJOR RSTUDIO_VERSION_MINOR RSTUDIO_VERSION_PATCH RSTUDIO_VERSION_SUFFIX;
+stdenv.mkDerivation rec {
```

Comment:
How about using `finalAttrs` pattern ?

Reply:
Thanks for the suggestion. This is still WIP so I'll put it into draft.